### ISSUE TITLE
bug: Empty `aws_acm_certificates` table from v19.1.0

### ISSUE THREAD

**Issue by akash1810** [cloudquery/cloudquery#12416]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

SInce v19.1.0 of the AWS source plugin, specifically https://github.com/cloudquery/cloudquery/pull/11761, ACM certificates are not collected.

Using the AWS CLI, I have 74 certificates:

```shell
➜ aws acm list-certificates --region eu-west-1 | jq -r '.CertificateSummaryList[].CertificateArn' | wc -l
      74
```

They are all RSA_2048:

```shell
➜ aws acm list-certificates --region eu-west-1 | jq -r '.CertificateSummaryList[].KeyAlgorithm' | uniq
RSA-2048
```

They are also all DNS validated, with a mixture of in use statuses.

However when running CloudQuery v19.1.0 and later, 0 items are collected.

Using any previous version yields the desired result.

### Expected Behavior

All ACM certificates should be collected, regardless of their key algorithm.

### CloudQuery (redacted) config

```yaml
kind: source
spec:
  name: aws
  path: cloudquery/aws
  version: "v19.1.0"
  tables: ["aws_acm_certificates"]
  destinations: ["file"]
  spec:
    regions:
      - eu-west-1
---
kind: destination
spec:
  name: "file"
  path: "cloudquery/file"
  version: "v3.3.0"
  spec:
    path: "data/{{TABLE}}/{{UUID}}.{{FORMAT}}"
    format: "csv"
    format_spec:
      delimiter: ","
```

### Steps To Reproduce

1. Create an ACM certificate
2. With the above config, run `sync --log-console --log-level debug`
3. Observe an empty `aws_acm_certificates` table

### CloudQuery (redacted) logs

```
2023-07-19T18:47:21Z INF Loading spec(s) args=["aws.yaml"] module=cli
2023-07-19T18:47:21Z INF Source plugin server listening address=/var/folders/2k/kh_57mss3254t3gx2qyt3_48___18m/T/cq-SKxGBNEMEVDUhwwv.sock module=cli
2023-07-19T18:47:22Z INF Plugin server listening address=/var/folders/2k/kh_57mss3254t3gx2qyt3_48___18m/T/cq-rNWAHlSfVzHJCHWO.sock module=cli
2023-07-19T18:47:23Z INF started call grpc.code=OK grpc.component=server grpc.method=GetVersions grpc.method_type=unary grpc.service=cloudquery.discovery.v0.Discovery grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.014 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF finished call grpc.code=OK grpc.component=server grpc.method=GetVersions grpc.method_type=unary grpc.service=cloudquery.discovery.v0.Discovery grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.038 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF started call grpc.code=OK grpc.component=server grpc.method=GetVersions grpc.method_type=unary grpc.service=cloudquery.discovery.v1.Discovery grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.016 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF finished call grpc.code=OK grpc.component=server grpc.method=GetVersions grpc.method_type=unary grpc.service=cloudquery.discovery.v1.Discovery grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.029 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF Start sync destinations=["file (v3.3.0)"] module=cli source="aws (v19.1.0)" sync_time=2023-07-19T18:47:23Z
2023-07-19T18:47:23Z INF started call grpc.code=OK grpc.component=server grpc.method=Init grpc.method_type=unary grpc.service=cloudquery.source.v2.Source grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.006 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF finished call grpc.code=OK grpc.component=server grpc.method=Init grpc.method_type=unary grpc.service=cloudquery.source.v2.Source grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.539 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF started call grpc.code=OK grpc.component=server grpc.method=GetDynamicTables grpc.method_type=unary grpc.service=cloudquery.source.v2.Source grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.005 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF finished call grpc.code=OK grpc.component=server grpc.method=GetDynamicTables grpc.method_type=unary grpc.service=cloudquery.source.v2.Source grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.126 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF started call grpc.code=OK grpc.component=server grpc.method=Configure grpc.method_type=unary grpc.service=cloudquery.destination.v1.Destination grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.007 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF finished call grpc.code=OK grpc.component=server grpc.method=Configure grpc.method_type=unary grpc.service=cloudquery.destination.v1.Destination grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.118 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF started call grpc.code=OK grpc.component=server grpc.method=Migrate grpc.method_type=unary grpc.service=cloudquery.destination.v1.Destination grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.003 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF finished call grpc.code=OK grpc.component=server grpc.method=Migrate grpc.method_type=unary grpc.service=cloudquery.destination.v1.Destination grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.114 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF End migration destinations=["file (v3.3.0)"] module=cli source="aws (v19.1.0)" time_took=0.000256
2023-07-19T18:47:23Z INF Start fetching resources destinations=["file (v3.3.0)"] module=cli source="aws (v19.1.0)"
2023-07-19T18:47:23Z INF started call grpc.code=OK grpc.component=server grpc.method=Sync grpc.method_type=server_stream grpc.service=cloudquery.source.v2.Source grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.046 module=cli peer.address= protocol=grpc
2023-07-19T18:47:23Z INF started call grpc.code=OK grpc.component=server grpc.method=Write grpc.method_type=client_stream grpc.service=cloudquery.destination.v1.Destination grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=0.191 module=cli peer.address= protocol=grpc
2023-07-19T18:47:24Z INF top level table resolver started client=000000000000:eu-west-1 module=aws-src table=aws_acm_certificates
2023-07-19T18:47:25Z INF table sync finished client=000000000000:eu-west-1 errors=0 module=aws-src resources=0 table=aws_acm_certificates
2023-07-19T18:47:25Z INF sync finished duration=1241.159959 errors=0 module=aws-src panics=0 resources=0
2023-07-19T18:47:25Z INF finished call grpc.code=OK grpc.component=server grpc.method=Sync grpc.method_type=server_stream grpc.service=cloudquery.source.v2.Source grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=2900.057 module=cli peer.address= protocol=grpc
2023-07-19T18:47:25Z INF finished call grpc.code=OK grpc.component=server grpc.method=Write grpc.method_type=client_stream grpc.service=cloudquery.destination.v1.Destination grpc.start_time=2023-07-19T19:47:23+01:00 grpc.time_ms=2900.596 module=cli peer.address= protocol=grpc
2023-07-19T18:47:25Z INF started call grpc.code=OK grpc.component=server grpc.method=Close grpc.method_type=unary grpc.service=cloudquery.destination.v1.Destination grpc.start_time=2023-07-19T19:47:25+01:00 grpc.time_ms=0.044 module=cli peer.address= protocol=grpc
2023-07-19T18:47:25Z INF finished call grpc.code=OK grpc.component=server grpc.method=Close grpc.method_type=unary grpc.service=cloudquery.destination.v1.Destination grpc.start_time=2023-07-19T19:47:25+01:00 grpc.time_ms=0.085 module=cli peer.address= protocol=grpc
2023-07-19T18:47:25Z INF End sync destinations=["file (v3.3.0)"] module=cli source="aws (v19.1.0)" sync_time=2023-07-19T18:47:23Z
2023-07-19T18:47:25Z INF Sending sync summary to analyticsv1.cloudquery.io:443 module=cli
2023-07-19T18:47:25Z INF started call grpc.code=OK grpc.component=server grpc.method=GetMetrics grpc.method_type=unary grpc.service=cloudquery.source.v2.Source grpc.start_time=2023-07-19T19:47:25+01:00 grpc.time_ms=0.018 module=cli peer.address= protocol=grpc
2023-07-19T18:47:25Z INF finished call grpc.code=OK grpc.component=server grpc.method=GetMetrics grpc.method_type=unary grpc.service=cloudquery.source.v2.Source grpc.start_time=2023-07-19T19:47:25+01:00 grpc.time_ms=0.151 module=cli peer.address= protocol=grpc
2023-07-19T18:47:26Z DBG sending interrupt signal to source plugin module=cli
2023-07-19T18:47:26Z INF waiting for source plugin to terminate module=cli
2023-07-19T18:47:26Z INF Got stop signal. Plugin server shutting down address=/var/folders/2k/kh_57mss3254t3gx2qyt3_48___18m/T/cq-rNWAHlSfVzHJCHWO.sock module=cli signal=interrupt
2023-07-19T18:47:26Z DBG sending interrupt signal to source plugin module=cli
2023-07-19T18:47:26Z INF waiting for source plugin to terminate module=cli
2023-07-19T18:47:26Z INF Got stop signal. Source plugin server shutting down address=/var/folders/2k/kh_57mss3254t3gx2qyt3_48___18m/T/cq-SKxGBNEMEVDUhwwv.sock module=cli signal=interrupt
```

### CloudQuery version

3.10.1

### Additional Context

I think https://github.com/cloudquery/cloudquery/pull/11761 caused the issue.

I'm attempting to raise a PR, but I'm not overly familiar with Go so it might take a little while!

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by erezrokah** [comment]
Thanks for reporting @akash1810 👍 I opened a PR to fix the issue [here](https://github.com/cloudquery/cloudquery/pull/12436)

**Comment by erezrokah** [comment]
The fix for this issue is released in [`v22.0.1`](https://github.com/cloudquery/cloudquery/releases/tag/plugins-source-aws-v22.0.1) of the AWS source plugin

**Comment by akash1810** [comment]
> The fix for this issue is released in [`v22.0.1`](https://github.com/cloudquery/cloudquery/releases/tag/plugins-source-aws-v22.0.1) of the AWS source plugin

Amazing. Thanks so much for this quick work @erezrokah! Confirming the issue is resolved in v22.0.1.
